draft-hoffman-dns-over-http-latest.mkd

```diff
@@ -145,7 +144,8 @@ In this document, the key words "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL
 "SHOULD", "SHOULD NOT", "RECOMMENDED", "MAY", and "OPTIONAL" are to be interpreted as
 described in BCP 14, RFC 2119 {{RFC2119}}.
 
-## Template
+Template
+=====
```

Comment:
Was the intent here to go to a top-level heading?  If so, just removing a `#` would have been fine, to keep things consistent.